src/components/Extensibility/components-form/CollapsibleRenderer.js

```diff
@@ -25,6 +25,7 @@ export function CollapsibleRenderer({
       title={tFromStoreKeys(storeKeys, schema)}
       defaultOpen={defaultOpen}
       nestingLevel={nestingLevel}
+      required={String(schema.get('required')) === 'true'}
```

Comment:
get this flag outside of this, like we have for `defaultOpen` for example, and probably you don't need thus `String()`